pipeline/dbt/models/intermediate/_models.yml

```
@@ -193,3 +193,20 @@ unit_tests:
         - {adresse_id: changed-code-insee}
         - {adresse_id: changed-commune}
         - {adresse_id: new}
+
+  - name: test_charges_null_is_really_null
+    model: int_dora__services
+    description: "Check if the NULL value in the stg file stays null and is not converted to {NULL}"
+    given:
+      - input: ref('stg_dora__services')
+        format: sql
+        fixture: stg_dora__services__input
```

intéressant ces fixtures :bulb: 

Mais pas convaincu qu'on préfère ça au dict inline pour un setup aussi simple :thinking: 

```suggestion
      - input: ref('stg_dora__services')
        rows:
          - {frais: '{gratuit}'}
```

1. on perd la possibilité de lire le test d'un seul coup de d'oeil. Il faut ouvrir 3 fichiers.
2. si la fixture est partagée comme tu l'imagines, on risque de se prendre la tête pour ne pas casser d'autres tests.
3. **imaginons** que le champ `frais` de dora puisse contenir `gratuit` et `{gratuit}` : on devrait ajouter un test case, 1 pour `gratuit` et un pour `{gratuit}`. Avec le dict inline, pas compliqué, il suffit d'ajouter une ligne. Avec la fixture "de base", ça va devenir beaucoup trop verbeux.

---

plein de réflexions pour un truc finalement assez simple, mais essentiel de s'accorder sur notre façon de procéder !

---

J'adorerais faire le dict inline mais cf ma reponse ci-dessus (les ARRAY posent souci, le JSON ne le poserait pas)